test/drenv/cache.py

```diff
@@ -0,0 +1,57 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import subprocess
+
+from . import commands
+
+
+def path(key):
+    cache_home = os.environ.get("XDG_CACHE_HOME", ".cache")
+    return os.path.expanduser(f"~/{cache_home}/drenv/{key}")
+
+
+def fetch(kustomization_dir, dest, log=print):
```

Comment:
I would prefer if fetch took a key and returned a path. If you have a strong reason to have it this way, I am willing to merge it as it is.

Reply:
We can do this, but we sill need to expose cache.path(). I was not sure about this change, so I kept this as 2 simple operations.

Lets add cache for few more addons and evaluate again.

Reply:
Ack